src/files-and-uploads/FileInfo.jsx

```diff
@@ -47,7 +80,7 @@ const FileInfo = ({
         <ModalDialog.Title>
           <div style={{ wordBreak: 'break-word' }}>
             <Truncate lines={2} className="font-weight-bold small mt-3">
-              {asset.displayName}
+              {asset?.displayName}
```

Comment:
This check that asset is not null seems to be quite a concern here, it looks like it will cause a bug if there is a check missing somewhere. So we need to safeguard against this breaking - the best way is to add a test where asset is null, and then we can actually have confidence that nothing will break here.